README.md

```diff
@@ -111,6 +111,9 @@ within that YAML file that corresponds to the scenario you wish to run. Defaults
 you must specify a host.
     E.g. `1.1.1.1`. Defaults to None.
 - `--influx_port`: Port for your `influx_host` in the case where it is non-default.
+- `--influx_ssl`: If your influxdb is using SSL, set this to True. Defaults to False.
+- `--influx_verify_ssl`: If your influxdb is using SSL, set this to True. Defaults to 
```

Comment:
Are these 2 different values or 2 different attributes? could one be False and one be True?
Could you be using ssl, but don't want the verify ssl to be true?

Reply:
Yes, they are two different attributes. It is possible for one to be False and the other to be True. This allows for flexibility in situations where you want to use SSL but don’t require certificate verification

Reply:
gotcha. thanks for clarifying.